volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -608,6 +616,32 @@ def get_vad_root(self):
             # windows xp and 2003
             return self.VadRoot.dereference().cast("_MMVAD")
 
+    def environment_variables(self):
+        """Generator for environment variables. 
+
+        The PEB points to our env block - a series of null-terminated
+        unicode strings. Each string cannot be more than 0x7FFF chars. 
+        End of the list is a quad-null. 
+        """
+        context = self._context
+        process_space = self.add_process_layer()
+
+        try:
+            block = self.get_peb().ProcessParameters.Environment
+            block_size = self.get_peb().ProcessParameters.EnvironmentSize
+            envars = context.layers[process_space].read(block, block_size).decode().split('\x00\x00\x00')[:-1]
+        except exceptions.InvalidAddressException:
+            return renderers.UnreadableValue()
+
+        for envar in envars:
+            split_index = envar.find('=')
+            env = envar[:split_index].replace('\x00', '')
+            var = envar[split_index+1:].replace('\x00', '')
+
+            # Exlude parse problem with some types of env
+            if env!= '' and var != '':
```

Comment:
These can be written as `if env and var`, which is neater and copes with things like `None`.